inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -128,6 +139,12 @@ Engine::LoadExeNetworkImpl(const InferenceEngine::ICNNNetwork &network, const st
         // valid for CNNNetworkImpl only, while there's no API in ICNNNetwork to change network
         ConstTransformer transformator(implNetwork.get());
         transformator.fullTrim();
+        NetPass::ConvertPrecision(*implNetwork, Precision::I64, Precision::I32);
+        NetPass::ConvertPrecision(*implNetwork, Precision::U64, Precision::I32);
+        NetPass::ConvertPrecision(*implNetwork, Precision::U32, Precision::I32);
+        NetPass::ConvertPrecision(*implNetwork, Precision::FP16, Precision::FP32);
+        NetPass::ConvertPrecision(*implNetwork, Precision::BOOL, Precision::U8);
+        NetPass::ConvertPrecision(*implNetwork, Precision::U16, Precision::I32);
```

Comment:
BTW, these passes will be called twice - on `ngraph::Function`, after it will be converted to old representation and passes will be called second time on old representation. May be we need to track that they are already called?

Reply:
what if we remove them completely? Some deprecated tests become to fail?

Reply:
The initial problem was that conversion to CNNNetwork overrides input/output precision with user precisions and then old ConvertPrecision passes was converting them. So after moving convert precision to ngrpah conversion to cnnnetwork became a problem because we do not convert precision after cnnnetwork conversion. So we decided to have dedicated transform that convert only input/output precision.

Reply:
And I can't just  take and remove old precision conversion because we have a lot of tests based on ir v7